services/slam/server_test.go

```diff
@@ -275,6 +296,18 @@ func TestFailingServer(t *testing.T) {
 		test.That(t, err.Error(), test.ShouldContainSubstring, "callback error")
 	})
 
+	t.Run("failing GetLatestMapInfo", func(t *testing.T) {
+
+		injectSvc.GetLatestMapInfoFunc = func(ctx context.Context) (time.Time, error) {
+			return time.Time{}, errors.New("failure to get latest map info")
+		}
+		reqInfo := &pb.GetLatestMapInfoRequest{Name: testSlamServiceName}
```

Comment:
[nit/opt] keep the pattern of using `req` and `resp` not `reqInfo` and `respInfo`

Reply:
See slack